TypeCobol.LanguageServer/TypeCobolServerHost.cs

```diff
@@ -191,7 +208,7 @@ static int Main(string[] args)
                 "",
                 "DESCRIPTION:",
                 "  Run the Language Server Robot.",
-                { "l|loglevel=",  "Logging level (1=Lifecycle, 2=Message, 3=Protocol).", (string v) =>
+                { "l|loglevel=",  "Logging level (0=Lifecycle, 1=Message, 2=Protocol).", (string v) =>
```

Comment:
Are we sure there is no other impact?
For instance, what about Language Server Robot Monitor?

Reply:
Fixing the usage message has most certainly no consequences. What matters however are the values used for the parameter. I've changed the level used during tests to match the correct value for `Protocol` both for the server and the robot. When the server (or the robot) is launched by our client we do not use the option and let the server (and the robot) use the default logging level.